lib/matplotlib/axes/_axes.py

```diff
@@ -471,8 +471,8 @@ def text(self, x, y, s, fontdict=None, withdash=False, **kwargs):
         return t
 
     @docstring.dedent_interpd
-    def annotate(self, *args, **kwargs):
-        a = mtext.Annotation(*args, **kwargs)
+    def annotate(self, text, xy, **kwargs):
```

Comment:
Looks like annotate can sometimes take a second xy (for the other end of the arrow)

Reply:
Whoops. Yeah, the explicit kwargs of the inner method could be passed as positional.